dipy/segment/select.py

```diff
@@ -0,0 +1,174 @@
+"""
+Select streamlines
```

Comment:
Not sure that we need a new module for this.
I think for standard processing for streamlines like this the right place is 
``dipy.tracking.streamline``